lib/core/pipes/user-initial.pipe.spec.ts

```diff
@@ -88,6 +89,17 @@ describe('UserInitialPipe', () => {
         expect(result).toBe('<div id="user-initials-image" class="">F</div>');
     });
 
+    it('should return a single letter from username if we do not have first and last name', () => {
+        const fakeIdentityUser: IdentityUserModel = {
+            firstName: undefined,
+            lastName: undefined,
+            username: 'FAKE-USERNAME'
+        };
+
+        const result = pipe.transform(fakeIdentityUser);
+        expect(result).toBe('<div id="user-initials-image" class="">F</div>');
```

Comment:
what if the username is lowercase ?

Reply:
Transform text to uppercase (for `firstName` and `lastName` also)

Reply:
ok my point is to have ```username: 'fake-username'```